rpm/gpsd.spec

```diff
@@ -84,22 +84,23 @@ scons %{_smp_mflags}          	\
     systemd=yes 		\
     debug=yes 			\
     leapfetch=no 		\
+    xgps=no             \
     python_libdir=%{python3_sitearch} \
     pkgconfigdir=%{_libdir}/pkgconfig
 
 # Fix python interpreter path.
 sed -e "s,#!/usr/bin/\(python[23]\?\|env \+python[23]\?\),#!/usr/bin/python3,g" -i \
-    gegps gpscat gpsfake xgps xgpsspeed gpsprof gps/*.py ubxtool zerk
+    gpsd-3.22/clients/gegps gpsd-3.22/clients/gpscat gpsd-3.22/gpsfake gpsd-3.22/clients/xgps gpsd-3.22/clients/xgpsspeed gpsd-3.22/clients/gpsprof gpsd-3.22/gps/*.py gpsd-3.22/clients/ubxtool gpsd-3.22/clients/zerk
```

Comment:
Please use %version here.

Reply:
Also using pushd and popd might make the command more simple

Reply:
Fedora spec doesn't have this hack any more and uses python_shebang=%{python3} parameter for the conscript build